ddtrace/encoding.py

```diff
@@ -4,12 +4,17 @@
 from .internal.logger import get_logger
 
 
-# Try to import msgpack, fallback to just JSON if something went wrong
+# check msgpack CPP implementation; if the import fails, we're using the
+# pure Python implementation that is really slow, so the ``Encoder`` should use
+# a different encoding format.
 # DEV: We are ok with the pure Python fallback for msgpack if the C-extension failed to install
 try:
-    from ddtrace.vendor import msgpack
-    # DEV: `use_bin_type` only exists since `0.4.0`, but we vendor a more recent version
-    MSGPACK_PARAMS = {'use_bin_type': True}
+    import msgpack
+    from msgpack._packer import Packer  # noqa
```

Comment:
please do not import underscore packages. This is what bite us last time, they renamed this internal package at one point and so the import failed and we were using JSON even though the C version of msgpack was available.

I think we should just use whatever version of msgpack (pure-Python or C) which is available to us via the normal/public interface.

As well, since msgpack published wheels, we'll likely always have the C-extension available to us.